src/i18n/fi/translation.json

```diff
@@ -205,7 +205,10 @@
     },
     "favorites": {
       "description": "Suosikeiksi tallentamasi koulutus- ja työmahdollisuudet löytyvät täältä. Voit tallentaa lisää mahdollisuuksia suosikeiksi kohtautuskoneen tulosnäkymässä. Voit poistaa suosikkeja sekä luoda yhdestä tai useammasta suosikista koulutus- ja urapolkusuunnitelman.",
-      "title": "Suosikkini"
+      "title": "Suosikkini",
+      "you-have-saved-n-opportunities_one": "Olet tallentanut yhden mahdollisuuden suosikiksi",
```

Comment:
Tried the plurals feature from i18next, I think it makes things a bit more user friendly. Any opinions about this?

Reply:
I find this great feature to be used 👍